.github/pr_path_labeler.yml

```diff
@@ -0,0 +1,46 @@
+# Labeler triggered by .github/workflows/labels.yml
+
+github:
+- any: [".github/**/*", "!.github/workflows"]
+
+CI:
+- any: ["**/*.yml", "**/*.yaml"]
```

Comment:
I suppose we could test if this works by not skipping the CI for this PR and seeing if the auto-label works to add `CI` here?

Reply:
I will try to push again. Since it's not merged, I was not sure it would work.

Reply:
No, it's not kicking in. I will try to merge on my fork and open a PR there to see. And actually, I am not sure we have a secret token on the repo, do you know?

@rgommers do we have `GITHUB_TOKEN` secret?

Reply:
Testing here. There is a bug, looking into it (one wrong path)
https://github.com/tupui/scipy/pull/2